lib/src/general/model/base_service.dart

```diff
@@ -31,9 +31,9 @@ abstract class BaseService {
   /// Constructor.
   BaseService(this._accessKey, {int timeout, List<String> features})
       : _timeout = (timeout == null) ? 5000 : timeout {
-    if (features?.contains('ENDABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX') ??
+    if (features?.contains('ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX') ??
         false) {
```

Comment:
Dart has some really nice features :) Having no truthy values feels like its for the best.